frontend/pages/DashboardPage/sections/PlatformHostCounts/_styles.scss

```diff
@@ -5,13 +5,23 @@
   // >= 320px and < 480px
   @media (min-width: $break-mobile-xs) {
     grid-template-columns: repeat(2, 1fr);
+
+    // For odd number of cards, make the first one span 2 columns
```

Comment:
Without this: between 320 and 480 px for odd number of cards, the last card was only half the width. Fixed so first card spans full width and rest of cards are paired.